crates/db_schema/src/lib.rs

```diff
@@ -175,6 +175,7 @@ pub enum SearchType {
   Communities,
   Users,
   Url,
+  Title,
```

Comment:
`PostTitle` would be a clearer name for this. And `PostUrl` as well, but that would be a breaking change.